lua/ulx/modules/sh/cfc_civilize.lua

```diff
@@ -0,0 +1,300 @@
+    ["stupid"] = "lacking in cognitive fortitude",
+    ["dumb"] = "mentally unencumbered",
+    ["noob"] = "novice enthusiast",
+    ["rekt"] = "thoroughly outmaneuvered",
+    ["kill yourself"] = "I suggest you reconsider your life choices",
+    ["stfu"] = "kindly cease your verbal emissions",
+    ["shut up"] = "I humbly request your silence",
+    ["fuck"] = "fornicate",
+    ["shit"] = "excrement",
+    ["damn"] = "drat",
+    ["crap"] = "unfortunate circumstance",
+    ["ass"] = "posterior",
+    ["bitch"] = "unpleasant individual",
+    ["bastard"] = "person of questionable lineage",
+    ["whore"] = "individual of negotiable affection",
+    ["slut"] = "person with a liberated lifestyle",
+    ["dick"] = "male anatomical reference",
+    ["pussy"] = "individual lacking fortitude",
+    ["wanker"] = "self-indulgent individual",
+    ["cunt"] = "person of disagreeable demeanor",
+    ["fag"] = "individual of alternative persuasion",
+    ["nigger"] = "person of African descent",
+    ["niggers"] = "people of African descent",
+    ["retard"] = "person with developmental challenges",
+    ["ugly"] = "aesthetically unremarkable",
+    ["fat"] = "gravitationally gifted",
+    ["skinny"] = "svelte",
+    ["gay"] = "homosexually inclined",
+    ["lesbian"] = "same-gender loving individual",
+    ["tranny"] = "transgender individual",
+    ["kys"] = "I urge you to seek professional assistance",
+    ["stfu"] = "please cease your endless rabble",
+    ["goon"] = "henchman",
+}
+
+local randomPhrases = {
+    "indubitably", "quite so", "indeed", "precisely", "absolutely", "certainly", "of course", "by all means",
+    "most assuredly", "without a doubt", "undoubtedly", "positively", "naturally", "clearly", "obviously",
+    "evidently", "apparently", "seemingly", "presumably", "supposedly", "allegedly", "reportedly", "ostensibly",
+    "I dare say", "I must confess", "if I may be so bold", "if you would be so kind", "as it were", "as one might expect",
+    "as the saying goes", "in any case", "in point of fact", "in all likelihood", "in the grand scheme of things",
+    "to be perfectly frank", "to put it mildly", "to say the least", "to my utmost delight", "to my great astonishment",
+}
+
+local function transform( sentence )
+    sentence = string.lower( sentence )
+
+    local replaceCount = 0
+
+    for word, replacement in pairs( transformations ) do
+        local currReplaceCount
+        sentence, currReplaceCount = string.gsub( sentence, "%f[%g]" .. word .. "%f[%G]", replacement )
+
+        replaceCount = replaceCount + currReplaceCount
+    end
+
+    local randomAddChance = 0.1
+    if replaceCount <= 1 then -- if no replacements were found, add lots of crap
+        randomAddChance = 0.5
+
+    end
+
+    local transformedWords = {}
+    for word in sentence:gmatch( "%S+" ) do
+        table.insert( transformedWords, word )
+        if math.random() < randomAddChance then -- Adjust the probability for inserting random phrases
```

Comment:
The probability isn't changing, we're just applying the probability.
```suggestion
        if math.random() < randomAddChance then
```